### DISCUSSION TITLE
这个错误是怎么解决的🥺 load_checkpoint_guess_config() got an unexpected keyword argument 'model_options'

### DISCUSSION THREAD

**Discussion by SANWU5** [yolain/ComfyUI-Easy-Use#323] [Q&A] [answered]
GPT的解释是 这个错误简化下来是：

在执行 easy fullLoader 时，程序调用了 load_checkpoint_guess_config 函数，但传递了一个名为 model_options 的参数，而这个函数并不接受该参数，导致程序报错。

关键点：

函数 load_checkpoint_guess_config 不支持 model_options 这个参数。
错误发生在 easyNodes.py 文件的 adv_pipeloader 方法中，具体是在加载检查点时调用了 load_checkpoint_guess_config。
可能的解决方案：

检查是否使用了错误版本的 ComfyUI 或者相关插件，可能新版本中 load_checkpoint_guess_config 函数的接口有所更改。
确保传递给函数的参数是它能够接受的。
如果这是自己编写或修改的代码，移除或更正 model_options 参数。
你可以尝试找到错误的 easyNodes.py 文件中的 adv_pipeloader 方法，并检查它调用 load_checkpoint_guess_config 时传递的参数，调整它们以匹配函数的实际接口。

**Comment by yolain** [comment, marked as answer]
更新comfyui

**Reply by SANWU5** [reply]
谢谢
